test/utils/AddressUtils.ts

```diff
@@ -137,18 +321,26 @@ describe('AddressUtils', async () => {
             'function fn () external view returns (bool)',
           ]);
 
-          await mock.mock.fn.returns(true);
+          //await mock.mock.fn.returns(true);
```

Comment:
This allows the test to pass because the call reverts for an unrelated reason.  For now, we will use a function on `AddressUtilsMock` as you mention in your comment below.

Reply:
See latest commit.